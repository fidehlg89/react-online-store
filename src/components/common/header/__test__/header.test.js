import React from "react";
import ReactDOM from "react-dom";
import Header from "..";

import { render, cleanup } from "@testing-library/react";
import renderer from "react-test-renderer"

afterEach(cleanup);

it("renders with out crashing", () => {
  const div=document.createElement('div');
  ReactDOM.render(<Header></Header>, div);
});

it("matches snapshot", ()=>{
  const tree=renderer.create(<Header></Header>).toJSON();
  expect(tree).toMatchSnapshot();
});
