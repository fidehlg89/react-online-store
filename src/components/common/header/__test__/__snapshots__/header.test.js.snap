// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<header
  className="sc-bdvvtL DHgkD"
  data-testid="header"
>
  <span>
     
  </span>
  <div>
    <img
      src="https://happytravel.viajes/wp-content/uploads/2020/04/146-1468479_my-profile-icon-blank-profile-picture-circle-hd.png"
      width="80px"
    />
  </div>
</header>
`;
